index.html

```diff
@@ -45,5 +45,149 @@ <h1>I’m Roodjino Glad to see you!</h1>
             </div>
         </section>
 
+        <section class="work">
+
+            <div class="card-work">
+                <div class="card-img">
+                    <img src="images/portfolio1.png" alt="snapshoot">
+                </div>
+                    
+                <div class="card-text">
+                    <h2>Project Title</h2>
+                    <ul class="project-details">
+                        <li>Canopy</li>
+                        <span class="bullet"></span>
+                        <li>Black end Dev</li>
+                        <span class="bullet"></span>
+                        <li>2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="project-stack">
+                        <li>HTML</li>
+                        <li>CSS</li>
+                        <li>JavaScript</li>
+                    </ul>
+                    <a href="#"> <button type="button">See Project</button> </a>
```

Comment:
Kindly add the hover, active, and pressed effects to the buttons to make your design conform to the actual figma design. :pray:

Reply:
Thank you for the Feedback! It's done